README.md

```diff
@@ -76,6 +76,7 @@ A `Dockerfile` is also available for your use - further information can be found
 
 Information about how to use the IoT Agent can be found in the
 [User & Programmers Manual](docs/usermanual.md).
+The following features are listed as [deprecated](https://github.com/telefonicaid/iotagent-json/tree/master/docs/deprecated.md)
```

Comment:
Please add a blank line just before this in order to have a line break in the document.

In addition, better to use a relative link for documentation and dot after it, i.e.

```
[deprecated](doc/deprecated.md).
```